automation/src/test/scala/org/broadinstitute/dsde/test/api/orch/RegistrationApiSpec.scala

```diff
@@ -54,5 +60,17 @@ class RegistrationApiSpec extends FreeSpec with Matchers with ScalaFutures with
       userInfo.userEmail should include (user.email)
       userInfo.enabled shouldBe true
     }
+
+    "should return terms of services with no auth token" in {
+      val req = HttpRequest(GET, ServiceTestConfig.FireCloud.orchApiUrl + s"tos/text")
+      val response = sendRequest(req)
+
+      val textFuture = Unmarshal(response.entity).to[String]
+
+      response.status shouldEqual StatusCodes.OK
+      whenReady(textFuture) { text =>
+        text should include("Terms as of February 12, 2020.")
```

Comment:
I am worried that a change to the ToS will break this test. Is there something else that we could assert that would be resilient to ToS changes?

Reply:
This makes sense. Truthfully, the only thing we know for sure is that the response status is OK. The assertion about the text contents is could always flake. The same issue is present in that SAM integration tests I added, which I can update-

Reply:
Could it test that the text is not empty? That seems like a safe thing for us to assume in the tests and it gives _some_ information

Reply:
Yes, it can, I'll test that